openupgrade_scripts/scripts/hr_recruitment/16.0.1.1/post-migration.py

```diff
@@ -0,0 +1,6 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "hr_recruitment", "16.0.1.1/noupdate_changes.xml")
```

Comment:
Also use the `delete_record_translations()` method for the corresponding fields (subject or content of the email template for example).